Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -563,14 +563,13 @@ body[dir=rtl] [dir=ltr] .go-to-arrow {
     margin: 6px 0 0;
 }
 
-.device-settings-content label + label {
+:is(.device-settings-content, .user-preferences-content) label + label {
```

Comment:
please move all `.user-preferences-content` styles to `Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.css` as that's where the corresponding DOM is created

i.e. the styles for `.device-settings-content` exist here because the corresponding DOM is created in `Source/WebInspectorUI/UserInterface/Base/Main.js`